Sources/MockoloFramework/Utils/SwiftSyntaxExtensions.swift

```diff
@@ -21,23 +54,330 @@ class EntityVisitor: SyntaxVisitor {
+        return Entity.model(label: "",
+                            name: node.name.text,
+                            typeName: node.inheritedType?.description ?? "",
+                            isGeneric: true,
+                            isInitializer: isInitializer,
+                            offset: node.offset,
+                            length: node.length)
+    }
+    
+    private func varModels(_ node: VariableDeclSyntax, acl: String, processed: Bool) -> [Model] {
+        // Detect whether it's static
+        var isStatic = false
+        if let modifiers = node.modifiers {
+            isStatic = hasStaticModifier(modifiers)
+        }
+
+        // Need to access pattern bindings to get name, type, and other info of a var decl
+        let varmodels = node.bindings.compactMap { (v: PatternBindingSyntax) -> Model? in
+            let name = v.pattern.firstToken?.text ?? String.unknownVal
+            var typeName = ""
+            var canBeInitParam = false
+            
+            // Get the type info and whether it can be a var param for an initializer
+            if let vtype = v.typeAnnotation?.type.description {
+                typeName = vtype
+                canBeInitParam = !isStatic &&
+                    !vtype.hasSuffix("?") &&
+                    !name.hasPrefix(.underlyingVarPrefix) &&
+                    !name.hasSuffix(.closureVarSuffix) &&
+                    !name.hasSuffix(.callCountSuffix) &&
+                    !name.hasSuffix(.subjectSuffix) &&
+                    vtype != .unknownVal
+            }
+            
+            let varmodel = Entity.model(name: name,
+                                         typeName: typeName,
+                                         acl: acl,
+                                         isStatic: isStatic,
+                                         canBeInitParam: canBeInitParam,
+                                         offset: v.offset,
+                                         length: v.length,
+                                         modelDescription: node.description,
+                                         processed: processed)
+            return varmodel
+        }
+        return varmodels
+    }
+    
+    private func memberList(_ members: MemberDeclListSyntax, overrides: [String: String]?, acl: String, processed: Bool) -> ([String], [Model], Bool) {
+        var attributeList = [String]()
+        var memberList = [Model]()
+        var hasInit = false
+        
+        for m in members {
+            if let varMember = m.decl as? VariableDeclSyntax {
+                let ret = varModels(varMember, acl: acl, processed: processed)
+                memberList.append(contentsOf: ret)
+                if let attrDesc = varMember.attributes?.withoutTrivia().description {
+                    attributeList.append(attrDesc.trimmingCharacters(in: .whitespacesAndNewlines))
```

Comment:
And `attributeList.append` can be at the end of the for loop too as it is common across all conditions with a different parameter. It can be further simplified by having a helper function which takes `m` and returns a <model, attribute> pair that can be appended in this loop.